workers/unity/Packages/com.improbable.gdk.mobile/Editor/MobileLauncherWindow.cs

```diff
@@ -0,0 +1,173 @@
+using System.Linq;
+using UnityEditor;
+using UnityEngine;
+
+namespace Improbable.Gdk.Mobile
+{
+    public class MobileLauncherWindow : EditorWindow
+    {
+        private const string MobileSectionLabel = "Mobile Settings";
+        private const string iOSSectionLabel = "iOS";
+        private const string AndroidSectionLabel = "Android";
+        private const string DevelopmentTeamIdLabel = "Development Team Id";
+        private const string RuntimeIpLabel = "Local runtime IP";
+        private const string ConnectLocallyLabel = "Should connect locally?";
+
+        private static readonly Color DarkGrey = new Color(0.4f, 0.4f, 0.4f, 1);
+        private static readonly Color LightGrey = new Color(0.7f, 0.7f, 0.7f, 1);
+
+        private MobileLaunchConfig launchConfig;
+
+        private string[] simulatorNames;
+        private string[] deviceNames;
+        
+        private int simulatorNameIndex;
+        private int deviceNameIndex;
+
+        [MenuItem("SpatialOS/Mobile launcher", false, 70)]
+        public static void ShowWindow()
+        {
+            var inspectorWindowType = typeof(EditorWindow).Assembly.GetType("UnityEditor.InspectorWindow");
+            var deploymentWindow = GetWindow<MobileLauncherWindow>(inspectorWindowType);
+            deploymentWindow.titleContent.text = "Mobile Launcher";
+            deploymentWindow.titleContent.tooltip = "A tab for managing your mobile client-workers.";
+            deploymentWindow.Show();
+        }
+
+        private void OnEnable()
+        {
+            launchConfig = new MobileLaunchConfig();
```

Comment:
Is it okay that `ShouldConnectLocally` is not persistent?

Reply:
yeah, I think it's ok. You only need it for launching the app which takes 2 seconds. was reluctant to create a new `json` file just for that